test1.py

```diff
@@ -464,14 +461,14 @@ def test_Component_27_T1(gg_util_obj, system_interface):
     # And kernel registered as a Thing
     # And my device is running the evergreen-kernel
     component_cloud_name = gg_util_obj.upload_component_with_version(
-        "HelloWorld", "1.0.0")
+        "HelloWorld", "1.0.0"
+    )
 
     # Give 5 sec for cloud to calculate artifact checksum and make it "DEPLOYABLE"
     time.sleep(5)
 
     # When I corrupt the contents of the component HelloWorld version 1.0.0 in the S3 bucket
-    assert gg_util_obj.upload_corrupt_artifacts_to_s3("HelloWorld",
-                                                      "1.0.0") == True
+    assert gg_util_obj.upload_corrupt_artifacts_to_s3("HelloWorld", "1.0.0") == True
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)

Similar issue at line number 488.